Sources/PointFree/Account/Account.swift

```diff
@@ -91,12 +91,12 @@ private let profileRowView = View<Database.User> { currentUser in
       div([
         h2([`class`([Class.pf.type.title4])], ["Profile"]),
 
-        form([action("#"), method(.post)], [
+        form([action(path(to: .updateProfile(nil))), method(.post)], [
           label([`class`([labelClass])], ["Name"]),
-          input([`class`([blockInputClass]), type(.text), value(currentUser.name)]),
+          input([`class`([blockInputClass]), name("name"), type(.text), value(currentUser.name)]),
```

Comment:
whoa, could we actually use `Database.User.CodingKeys.name.rawValue` here?

Reply:
Yep! Just Gott make them public. Nested keys we need another solution for, tho.